http3/response_writer.go

```diff
@@ -3,35 +3,49 @@ package http3
 import (
 	"bufio"
 	"bytes"
-	"io"
 	"net/http"
 	"strconv"
 	"strings"
 
+	"github.com/lucas-clemente/quic-go"
 	"github.com/lucas-clemente/quic-go/internal/utils"
 	"github.com/marten-seemann/qpack"
 )
 
+// DataStreamer lets the caller take over the stream. After a call to DataStreamer
+// the HTTP server library will not do anything else with the connection.
```

Comment:
is connection the best term here?

Reply:
That was copied from `http.Hijacker` ;)
Of course it's the stream here.